svgnative/CMakeLists.txt

```diff
@@ -124,7 +124,7 @@ elseif(MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /we4302 /we4189")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4290 /wd4335 /wd4355 /wd4814 /wd4091 /TP")
     if (NOT USE_SKIA)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++11")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++14")
```

Comment:
The idea is to have an interoperable library that is able to go back to C++11. So a requirement for C++14 should be avoided.

We do have code that allows to switch to C++17 but keeping C++11 compatibility. See PR #194.